src/uiprotect/api.py

```diff
@@ -1569,6 +1713,65 @@
         r.close()
         return None
 
+    async def get_camera_video(
+        self,
+        camera_id: str,
+        start: datetime,
+        end: datetime,
+        channel_index: int = 0,
+        validate_channel_id: bool = True,
+        output_file: Path | None = None,
+        iterator_callback: IteratorCallback | None = None,
+        progress_callback: ProgressCallback | None = None,
+        chunk_size: int = 65536,
+        fps: int | None = None,
+        filename: str | None = None,
+    ) -> bytes | None:
+        """
+        Deprecated: maintained for backwards compatibility.
+
+        If you are using Unifi Protect 4 or later, please use
+        prepare_camera_video() and download_camera_video() instead.
+        """
```

Comment:
Thats probably a good idea, but we can't do late imports in async code because it does blocking I/O

@coderabbitai remember we can't do late imports in async code because it does blocking I/O

Reply:
@coderabbitai please produce a committable suggestion

Reply:
Do we really want to depreciate this? I just took the code from the old PR, but there is nothing wrong with calling this function per se. 

I'm kinda dubious about doing this behind the scene switcheroo in the first place. I don't want to introduce some unexpected bug into downstream dependencies